docs/source/install.rst

```diff
@@ -48,12 +48,12 @@ the binaries and images.
 .. note:: If you want a specific release, pass a version identifier for Fabric,
           Fabric-ca and thirdparty Docker images.
           The command below demonstrates how to download the latest production releases -
-          **Fabric v2.1.1** and **Fabric CA v1.4.7**
+          **Fabric v2.2** and **Fabric CA v1.4.7**
 
 .. code:: bash
 
   curl -sSL https://bit.ly/2ysbOFE | bash -s -- <fabric_version> <fabric-ca_version> <thirdparty_version>
-  curl -sSL https://bit.ly/2ysbOFE | bash -s -- 2.1.1 1.4.7 0.4.20
+  curl -sSL https://bit.ly/2ysbOFE | bash -s -- 2.2 1.4.7 0.4.20
```

Comment:
We intend to release 2.2 next week. 
Therefore either this change should be removed from the PR, or we simply sit on this PR for a week. Either way is fine by me...

Reply:
I'll remove this for now so we can do a dedicated PR just for this.